jwql/website/apps/jwql/templates/base.html

```diff
@@ -76,7 +76,7 @@
 	            		<a class="nav-link" href={{ url('jwql:edb') }}>EDB<span class="sr-only">(current)</span></a>
 	          		</li>
 		      		<li class="nav-item active">
-	            		<a class="nav-link" href="https://jwql.readthedocs.io/en/latest/">Documentation<span class="sr-only">(current)</span></a>
+	            		<a class="nav-link" href="https://jwql.readthedocs.io/en/latest/" target="_blank" rel="noopener noreferrer">Documentation<span class="sr-only">(current)</span></a>
```

Comment:
@cracraft First time i've seen `rel="noopener noreferred"`, I looked it up and it seems like a good idea!  Cool!